docs/source/interfaces/DataProcessorWidget.rst

```diff
@@ -0,0 +1,751 @@
+| GenericDataProcessorPresenter       |                                                          |
++-------------------------------------+----------------------------------------------------------+
+| DataProcessorMainPresenter          |                                                          |
++-------------------------------------+----------------------------------------------------------+
+| DataProcessorPreprocessingAlgorithm |                                                          |
++-------------------------------------+----------------------------------------------------------+
+| DataProcessorProcessingAlgorithm    |                                                          |
++-------------------------------------+----------------------------------------------------------+
+| DataProcessorPostprocessingAlgorithm|                                                          |
++-------------------------------------+----------------------------------------------------------+
+| DataProcessorTreeManager            |                                                          |
++-------------------------------------+----------------------------------------------------------+
+| DataProcessorOneLevelTreeManager    |                                                          |
++-------------------------------------+----------------------------------------------------------+
+| DataProcessorTwoLevelTreeManager    |                                                          |
++-------------------------------------+----------------------------------------------------------+
+| QDataProcessorOneLevelTreeModel     |                                                          |
++-------------------------------------+----------------------------------------------------------+
+| QDataProcessorTwoLevelTreeModel     |                                                          |
++-------------------------------------+----------------------------------------------------------+
+| DataProcessorCommand                |                                                          |
++-------------------------------------+----------------------------------------------------------+
+| DataProcessorCommandBase            |                                                          |
++-------------------------------------+----------------------------------------------------------+
+| DataProcessor***Command             |                                                          |
++-------------------------------------+----------------------------------------------------------+
+| DataProcessorCommandAdapter         |                                                          |
++-------------------------------------+----------------------------------------------------------+
+| DataProcessorGenerateNotebook       |                                                          |
++-------------------------------------+----------------------------------------------------------+
```

Comment:
There are no descriptions of these classes. Should there be descriptions or has it been left blank on purpose?